tools/gh_api.py

```diff
@@ -47,7 +47,7 @@ def get_auth_token():
         with open(os.path.join(os.path.expanduser('~'), '.ghoauth')) as f:
             token, = f
             return token
-    except:
+    except OSError:
```

Comment:
Oh, I missed the fact that the two are not synonyms in Py2 :-)  Can `open` raise both `OSError` and `IOError`?